Apps/WebAPI/sdLitica.WebAPI/Helpers/AppSettings.cs

```diff
@@ -0,0 +1,9 @@
+namespace sdLitica.Helpers
+{
+    public class AppSettings
```

Comment:
Please, move this class to the namespace ``sdLitica.Utils.Helpers`` in the ``sdLitica.Utils`` project. This Utils project I have already created. Wait for my Merge.